src/NzbDrone.Core/ImportLists/Simkl/User/SimklUserRequestGenerator.cs

```diff
@@ -21,7 +23,7 @@ public virtual ImportListPageableRequestChain GetListItems()
 
         private IEnumerable<ImportListRequest> GetSeriesRequest()
         {
-            var link = $"{Settings.BaseUrl.Trim()}/sync/all-items/shows/{((SimklUserListType)Settings.ListType).ToString().ToLowerInvariant()}";
+            var link = $"{Settings.BaseUrl.Trim()}/sync/all-items/{ShowType}/{((SimklUserListType)Settings.ListType).ToString().ToLowerInvariant()}";
```

Comment:
When coming from settings this will need to convert the enum to a string, but a ternary can handle that while there are only two values.

Something like:
`var showType = Settings.ShowType == (int)SimklShowType.Anime ? 'anime' : 'shows'`